docs/sources/enterprise/activate-license.md

```diff
@@ -1,4 +1,9 @@
-
++++
+title = "Activate an Enterprise license"
+description = "Activate an Enterprise license"
+keywords = ["grafana", "licensing"]
```

Comment:
Do we keyword enterprise?

Reply:
I have not seen consistency, in some pages we did, in some we did not. But I think it makes sense, I am going add it.

Reply:
Approved because this PR corrects the missing menu item. Terminology wrt GE can be addressed in a subsequent PR.